src/openstudio_lib/RunTabView.cpp

```diff
@@ -490,31 +490,29 @@ void RunView::readyReadStandardOutput() {
 
     QString trimmedLine = line.trimmed();
 
-    bool b = trimmedLine.contains("DEBUG");
-
     // DLM: coordinate with openstudio-workflow-gem\lib\openstudio\workflow\adapters\output\socket.rb
     if (trimmedLine.isEmpty()) {
       continue;
     } else if ((trimmedLine.contains("DEBUG")) || (trimmedLine.contains("] <-2>"))) {
       m_textInfo->setFontPointSize(10);
       m_textInfo->setTextColor(Qt::lightGray);
-      m_textInfo->append(line);
+      m_textInfo->append(trimmedLine);
```

Comment:
https://doc.qt.io/qt-6/qtextedit.html#append

> void QTextEdit::append(const [QString](https://doc.qt.io/qt-6/qstring.html) &text)

> Appends a new **paragraph** with text to the end of the text edit.

So trimmedLine is perfectly good here.

I tested on Windows and it fixes the issue. I tested on mac that it didn't create new problems.

Reply:
Awesome!